### PR TITLE
Feature disable banner

### PR DESCRIPTION
**Pull request by costastf**

### CODE REVIEW

File: awsenergylabelercli/awsenergylabelercli.py [hunk 1]
```diff
@@ -230,6 +230,12 @@ def get_parser():
                         default=environment_variable_boolean(os.environ.get('AWS_LABELER_DISABLE_SPINNER',
                                                                             False)),
                         help='If set spinner will be disabled on the CLI.')
+    parser.add_argument('--disable-banner',
+                        '-db',
+                        action='store_true',
+                        default=environment_variable_boolean(os.environ.get('AWS_LABELER_DISABLE_SPINNER',
```
Comment (jendab) [inline awsenergylabelercli/awsenergylabelercli.py:236]: environment var should be `AWS_LABELER_DISABLE_BANNER`

File: requirements.txt [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
 # Please use Pipfile to update the requirements.
 #
-coloredlogs~=10.0
-awsenergylabelerlib~=3.1.0
-yaspin~=2.1.0
-terminaltables~=3.1.10
-art~=5.7
\ No newline at end of file
+coloredlogs>=15.0.1
+awsenergylabelerlib>=3.2.0
```
Comment (jendab) [inline requirements.txt:11]: Was this updated? The Pipfile shows `awsenergylabelerlib = ">=3.2.0,<4.0"`

File: awsenergylabelercli/awsenergylabelercli.py (hunk 1 above)
Comment (jendab) [reply awsenergylabelercli/awsenergylabelercli.py]: Fixed

Comment (costastf) [reply awsenergylabelercli/awsenergylabelercli.py]: Weird, I have tested the feature before pushing, both with args and env vars. Not sure how this happened.

**Pull request merged**